pkg/controllers/store_controller.go

```diff
@@ -84,12 +84,12 @@ func (r *StoreReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl
 // SetupWithManager sets up the controller with the Manager.
 func (r *StoreReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	return ctrl.NewControllerManagedBy(mgr).
-		For(&configv1alpha1.Store{}).
+		For(&configv1beta1.Store{}).
 		Complete(r)
 }
 
 // Creates a store reference from CRD spec and add store to map
-func storeAddOrReplace(spec configv1alpha1.StoreSpec, fullname string) error {
+func storeAddOrReplace(spec configv1beta1.StoreSpec, fullname string) error {
```

Comment:
i don't see usage of unversioned. if it's not Kubernetes API reference, you should use unversioned

Reply:
Actually in controller, the `Store` object is retrieved from K8s Cluster which should be a valid version. https://github.com/deislabs/ratify/blob/main/pkg/controllers/store_controller.go#L64
So I use the v1beta1 version across the controller file, for other files we should definitely use unversioned API but there is no usage yet. 
And for this specific function, we could probably do a manual conversion from v1beta1 to unversioned once we retrieved object from the cluster.